astroid/nodes/scoped_nodes.py

```diff
@@ -515,6 +518,7 @@ def __init__(
         self.parent = parent
         self.pure_python = pure_python
         self.locals = self.globals = {}
+        self.doc_node = doc_node
```

Comment:
`doc_node` should be added to `_astroid_fields` above as it can contain a astroid node.